Documentation/test-suites-overview.md

```diff
@@ -0,0 +1,228 @@
+# Mender QA test suites overview
+
+This document gives an overview of all the different tests suites that we have
+in Mender
+
+## Introduction: test levels
+
+At mender we have different test suites that follow the Development Lifecycle,
+verifying our software at different levels through the different phases of the
+Development Lifecycle.
+
+Due to our microservices architecture, we name our test suites in a different
+way than standards defined by ISTQB (International Software Testing
+Qualifications Board) or other online references. See (ISTQB Foundation Level
+Syllabus)[https://www.istqb.org/downloads/syllabi/foundation-level-syllabus.html].
+
+Nevertheless, we follow QA best practices and principles in verifying our
+software product and it's individual components at different levels.
+
+## Introduction: programming languages
+
+With the exception of the GUI and some client addons in C, the core components
+of Mender are written in Go.
+
+The unit level tests are written i the same language as the test under code
+(usually Go), while for every other test suite we use Python programming
+language and Pytest framework.
+
+## unit tests
+
+Test suite       | unit tests
+---------------- | ----------
+ISTQB equivalent | Component testing
+Test object      | The code inside a single repository
+Source code      | Repository specific, all `*_test.go` files
+How to run       | `go test $(go list ./...)`
```

Comment:
I think `go test ./...` suffices, doesn't it?

Reply:
Yes, you are right. Thanks.